README.md

```diff
@@ -50,3 +50,98 @@ isAuthorized            | boolean        | `true`                    | When true
 routes                  | Route[]        |                           | The list of inner routes
 requiredPermissions     | string[]       | `['cdn_servers', 'edit']` | The list can be used inside `isPermittedRoute` function in order to filter not permitted routes
 redirect                | boolean        | `true`                    | When true, `<Redirect />` will be rendered. Works together with `pathTo`
+
+## Config
+
+Allow to create common config object
+```javascript
+import { Config } from '@outcast.by/js-ext'
+```
+
+### Initialization
+1. Create config file
+    ```javascript
+    // config.js
+    
+    import jsExt from './common/jsExt'
+    
+    export default {
+      jsExt,
+      cookiesExpires: 14, // days
+      secretKey: process.env.REACT_APP_SECRET_KEY,
+    }
+    ```
+2. Create initializer for config and start it with application
+    ```javascript
+    import { Config } from '@outcast.by/js-ext'
+    import config from '../config'
+    
+    export default function () {
+      Config.set(config)
+    }
+    ```
+### Get value from config
+Config.get(path, defaultValue)
+
+Example:
+```javascript
+Config.get(['jsExt', 'secretKey'], '')
+```
+
+### Add new value to config after initialization
+Config.setIn(path, value)
+Example:
+```javascript
+Config.setIn(['jsExt', 'cookiesExpires'], 10)
+```
+
+## Utils
+
+### graphql
+Create config file for jsExt:
+```javascript
+import axios from 'axios'
+
+export default {
+  axios,
+  url: `${process.env.REACT_APP_SERVER_URL}/public`,
+  options: { withCredentials: true },
+}
+```
+It is necessary to add `axios` to config for using initialized in app `axios`.
```

Comment:
We have to add `axios` in we wanna use global headers, interceptors, etc inside the main app